### PR TITLE
fix: small leftover issues with new starter screen

### LINKED ISSUE #3642: bug: Some issues with Jan's new starter screen
### Jan version

0.5.3-629

### Describe the Bug

- [x] 1. Change starter models to: **Gemma 2 2B Q4** and **Llama 3 8B Q4**
- [x] 2. Improve current responsive for cloud model section: 

https://github.com/user-attachments/assets/90bc2a0e-15de-47af-9326-8826abbfe220

- [ ] 3. Add size next to on-device models
Figma: https://www.figma.com/design/DYfpMhf8qiSReKvYooBgDV/Jan-App-(3rd-version)?node-id=4270-69850&t=af5Wyb9D89fwRirc-4
Screenshot 2024-09-12 at 1 50 17 PM

### LINKED ISSUE #3660: bug: Small leftover issues with Jan's new Starter Screen
### Jan version

0.5.3-635

### 1. We should disable the "New Threads" icon on the top nav when the user is in the Starter Screen
Screenshot 2024-09-14 at 15 51 16

### 2. It shows two downloading statuses when I click to download two models 
Screenshot 2024-09-14 at 15 41 52

### 3. The model recommendation is different from machine to machine
e.g, on @urmauur's side it shows: Gemma 2 + Llama 3.1; on @imtuyethan's side it shows: Gemma 2 + Hermes Pro 7B

### 4. It caches the download status when I use the search bar
Screenshot 2024-09-14 at 16 01 16

### 5. Cloud model should be left aligned 
Otherwise, when the 2nd row has only 1 model, it'd look weird
Screenshot 2024-09-14 at 16 02 22

### 6. Some other tiny UI issues
Screenshot 2024-09-14 at 15 52 52

### What is your OS?

- [X] MacOS
- [ ] Windows
- [ ] Linux

### PR DESCRIPTION
**Pull request by urmauur**
## Describe Your Changes

1. Disable the "New Threads" icon on the top
image

2. Duplicate render progress bar when more than one download from feat/suggestion model.
Screenshot 2024-09-14 at 23 01 20

3. @louis-jan temporary with hard-coded recommend model

4. Overlap progress bar when dropdown search open

image

5. Cloud model should be left aligned
image

6. Update UI regarding miss font weight and spacing
image

### Changes code:

1. Update conditional render `New threads` icon when user downloads a local model or configured any cloud model and thread length 
Screenshot 2024-09-15 at 00 34 40

2. The new code filters the `downloadStates` object values using the condition `x.modelId === featModel.id`. This means that only items with a `modelId` matching `featModel.id` will be rendered.
Screenshot 2024-09-14 at 23 24 14

3. temporary with hard-coded recommend model 

4. Update `z-index` to avoid overlap, since the element dropdown options is absolute 
Screenshot 2024-09-15 at 00 42 34

5. Removed justify center flex 
 Screenshot 2024-09-15 at 00 47 24

6. Change some class font weight and spacing
Screenshot 2024-09-15 at 01 01 35

## Fixes Issues

- Closes #3660  
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

**Review by dan-homebrew — APPROVED** [review]
Approving to unblock, and will let @imtuyethan @eckartal find issues during QA process

**Review by dan-homebrew — APPROVED** [review]
Approving to unblock. Will let @imtuyethan @eckartal find issues during Manual QA

**Comment by dan-homebrew** [comment]
@urmauur Can you look into why the Playwright tests are failing?

**Comment by urmauur** [comment]
> @urmauur Can you look into why the Playwright tests are failing?

Hi @dan-homebrew and @louis-jan,
Can you check again this PR?

The issue with the PR failure is in the test `Select GPT model from Hub and Chat with Invalid API Key`. Since we updated the thread with the starter screen, the “Create Thread” button is no longer rendered, so the `btn-create-thread` selector is missing.

However, the `btn-create-thread` selector is not necessary because the test scenario follows this flow:

- Go to the hub page
- Select cloud mode
- Send a chat
- Expect the `APIKeyError` component to render
	
removed this code 

```tsx
await page
    .getByTestId('btn-create-thread')
    .click()
```

**Comment by urmauur** [comment]
> @urmauur Can you look into why the Playwright tests are failing?

Hi @dan-homebrew and @louis-jan,
Can you check again this PR?

The issue with the PR failure is in the test `Select GPT model from Hub and Chat with Invalid API Key`. Since we updated the thread with the starter screen, the “Create Thread” button is no longer rendered, so the `btn-create-thread` selector is missing.

However, the `btn-create-thread` selector is not necessary because the test scenario follows this flow:

- Go to the hub page
- Select cloud mode
- Send a chat
- Expect the `APIKeyError` component to render
	
removed this code 

```tsx
await page
    .getByTestId('btn-create-thread')
    .click()
```

File: web/screens/Thread/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```
Comment (urmauur) [inline web/screens/Thread/index.test.tsx:1]: Add a unit test here to ensure the thread screen renders correctly

File: electron/tests/e2e/thread.e2e.spec.ts [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
+test('Select GPT model from Hub and Chat with Invalid API Key', async ({
+  hubPage,
+}) => {
   await hubPage.navigateByMenu()
   await hubPage.verifyContainerVisible()
 
   // Select the first GPT model
   await page
     .locator('[data-testid^="use-model-btn"][data-testid*="gpt"]')
-    .first().click()
```
Comment (urmauur) [inline electron/tests/e2e/thread.e2e.spec.ts:11]: Removed unnecessary selector

File: electron/tests/e2e/thread.e2e.spec.ts [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
-
-  // Attempt to create thread and chat in Thread page
-  await page
-    .getByTestId('btn-create-thread')
```
Comment (urmauur) [inline electron/tests/e2e/thread.e2e.spec.ts:15]: Removed unnecessary selector

**Review by louis-jan — APPROVED** [review]
LGTM

Just need to find somehow to define recommended models better.

**Pull request merged**